ref/Microsoft.Build/net/Microsoft.Build.cs

```diff
@@ -1,5 +1,12 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
+namespace Microsoft.Build.BackEnd
```

Comment:
We haven't exposed this namespace before, so I think we should move the exception into the `Microsoft.Build.Exceptions` namespace where the others are:

https://github.com/dotnet/msbuild/blob/dcf8be378c86c0fff1dea57e0b532f322f09d191/ref/Microsoft.Build/net/Microsoft.Build.cs#L887-L890

Can you change the namespace in `CircularDependencyException.cs`?

Reply:
sounds good! I think keeping it protected was causing some error preventing it from building. The build worked on my end, I think the namespace change might resolve that, so I changed it back to protected.